projects/aws/eks-anywhere-test/Makefile

```diff
@@ -17,8 +16,12 @@ RELEASE_TARGETS=images
 
 include $(BASE_DIRECTORY)/Common.mk
 
+copy-attribution:
```

Comment:
Why the empty attribution? If has_licenses was true then this additional copy Target wouldn't be necessary since it automatically does this before building images. That wouldn't work tho since there is no go code for it to try and gather licenses from.

Reply:
I couldn't get attributions to work. Maybe I should remove this and create a ticket to work on that later

Reply:
What wouldn't work? Generating one or was it complaining somewhere that it didn't exist? It can't really generate one since it depends on using go-licenses to gather licenses from a go source tree which this project doesn't have.

Reply:
I made attributes optional for now and this works. I created an issue to fix attributions for this project. I really want these changes for *other* projects.

Out of the box the attribution generation fails trying to checkout the GIT_TAG of the non-existent repo

Reply:
Right, attribution generation is dependent on a golang source project, typically from cloning upstream.  I think I see what you were trying to do now with the attribution file in the helm script.  I have a thought Ill try out and see how it looks.